plutus-pab/src/Plutus/PAB/Webserver/Handler.hs

```diff
@@ -202,34 +181,38 @@ parseContractId t =
         Nothing   -> throwError $ InvalidUUIDError t
 
 handler ::
-       forall effs.
-       ( Member (EventLogEffect (ChainEvent ContractExe)) effs
-       , Member (ContractEffect ContractExe) effs
+       forall effs m appBackend.
+       ( Member (ContractEffect ContractExe) effs
        , Member ChainIndexEffect effs
-       , Member MetadataEffect effs
        , Member UUIDEffect effs
-       , Member (LogMsg LM.ContractExeLogMsg) effs
        , Member (Error PABError) effs
        , Member (LogMsg UnStringifyJSONLog) effs
        , Member (LogMsg (Instance.ContractInstanceMsg ContractExe)) effs
+       , Instance.AppBackendConstraints ContractExe m appBackend
+       , LastMember m effs
+       , Member (Reader InstancesState) effs
+       , Member (ContractDefinitionStore ContractExe) effs
+       , Member (ContractStore ContractExe) effs
+       , LastMember m (Reader ContractInstanceId ': appBackend)
```

Comment:
Shouldn't this constraint be the same as `LastMember m appBackend`. Just curious as to why you need to prepend `ContractsInstanceId`.